notebooks/244-depth-estimation-videpth/244-depth-estimation-videpth.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>please provide some descriptions, transcripts or/and links for used abbreviations (VIO, VOID, IMU, SML)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/978/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>The first usage of such abbreviations has now been updated with the complete form. However, the spell-checking action has not been up to the mark. Proper nouns are being detected as errors such as names of the authors and the name of the camera used for the collection of the data alongside common words like odometry, accelerometer etc. File extensions such as <code>.ckpt</code> and a small section to introduce the concept of model naming conventions to the readers (which has many such similar words) have also been caught as errors. </p>